docker-magento2/.docker/bin/fix-permission

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+chmod -R 777 /var/www/
+# find /var/www/html/ -type f -exec chmod 644 {} \;      # 644 permission for files
```

Comment:
Don't commit commented out code. (unless there is a very good reason, but no good reason is explained here)